doubleml/double_ml_blp.py

```diff
@@ -0,0 +1,195 @@
+import statsmodels.api as sm
+import numpy as np
+import pandas as pd
+
+from scipy.stats import norm
+from scipy.linalg import sqrtm
+
+
+class DoubleMLBLP:
+    """Best linear predictor (BLP) for DoubleML with orthogonal signals.
+
+        Parameters
+        ----------
+        orth_signal : :class:`numpy.array`
+            The orthogonal signal to be predicted. Has to be of shape (n,).
```

Comment:
I'd recommend to specify what `n` is here. I guess, users might wander if the `n` has to match some value, like the number of observations in the IRM model or so... Maybe it's good to briefly explain this in a sentence